deploy.sh

```diff
@@ -87,6 +54,13 @@ then
 fi
 rm dist/*.bak
 
+# get the config file and repackage to redeploy
+$WSK action get "${ACTION_NAME}" --save --apihost adobeioruntime.net --auth "${ADOBE_RUNTIME_AUTH}"
+mkdir previous
+unzip "${ACTION_NAME}.zip" -q -d previous
+cp previous/config.json dist/.
+rm -rf "${ACTION_NAME}.zip"
```

Comment:
```suggestion
rm -rf "${ACTION_NAME}.zip" previous/
```

Reply:
sure, the only reason i removed the zip is because the name conflicted with the deployment zip we're about to generate

Reply:
Oh, okay. I thought we were cleaning up :)